src/Symfony/Component/Cache/composer.json

```diff
@@ -31,11 +31,14 @@
         "doctrine/cache": "~1.6",
         "doctrine/dbal": "~2.5",
         "predis/predis": "~1.0",
-        "symfony/var-dumper": "^4.1.1"
+        "symfony/var-dumper": "^4.1.1",
+        "symfony/dependency-injection": "~4.2",
+        "symfony/config": "~4.2"
     },
     "conflict": {
         "doctrine/dbal": "<2.5",
-        "symfony/var-dumper": "<3.4"
+        "symfony/var-dumper": "<3.4",
+        "symfony/dependency-injection": "<4.2"
```

Comment:
not sure this is required, is it? 3.4 could be fine to me (same for "require-dev")